www/source/docs/reference.html.md.erb

```diff
@@ -0,0 +1,46 @@
+---
+title: Reference overview
+---
+
+# Habitat Reference and Syntax
+
+When defining a package in habitat, there are several different files, settings, variables, and functions that you can use to set up specific installation and configuration details. Because packages are simply scripts and templated files, you have a lot of flexibility in how you build and define the configuration of your application.
+
+This section will cover as many of the modules and APIs that Habitat is built upon to serve as helpful reference as you're developing your habitat packages.
+
+## Table of Contents
+
+- [Environment Variables](#environment-variables)
+- [Plan Settings](#plan-settings)
+- [Plan Variables](#plan-variables)
+- [Plan Helper Functions](#utility-functions)
+- [Callbacks](#reference-callbacks)
+- [Lifecycle Hooks](#reference-hooks)
```

Comment:
Call these "Supervisor Application Lifecycle Hooks" to make it clear these are related to the Supervisor, not the Builder